### PR TITLE
Added makefile

### PR DESCRIPTION
**Pull request by DavidLu1997**
Added makefile and made webapp build

@Denton-L looks good?

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -0,0 +1,41 @@
+BUILD_WEBAPP_DIR = ./webapp
+
+check-style:
```
Comment (Denton-L) [inline makefile]: Why are your commands each spaced so far apart?

Comment (DavidLu1997) [inline makefile]: I like spaces

Comment (Denton-L) [inline makefile]: I feel like it should be something like this:

```
build:
    command1
    command2
    command3

other-stuff:
    command1
    command2
    command3

File: makefile [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	@echo Checking style
+
+	gofmt -w .
+
+	cd $(BUILD_WEBAPP_DIR) && $(MAKE) check-style
+
+build:
```
Comment (Denton-L) [inline makefile]: I feel like this should be moved to the top. That way, people can type in `make` and it'll default to building and not checking the style.

Comment (DavidLu1997) [inline makefile]: order changed

**Pull request merged**